libraries/AP_Param/AP_Param.cpp

```diff
@@ -2083,6 +2083,15 @@ void AP_Param::convert_class(uint16_t param_key, void *object_pointer,
             idx = 63;
         }
 
+        if (info.type == AP_PARAM_GROUP) {
+            // Covert subgroups if enabled
+            if (recurse_sub_groups) {
+                // Only recurse once
+                convert_class(param_key, (uint8_t *)object_pointer + group_info[i].offset, get_group_info(group_info[i]), idx, false, false);
+            }
+            continue;
```

Comment:
maybe an assert in SITL if we ever are skipping a group without recurse_sub_groups? is there any time that is valid?